From: Ryusuke Konishi <konishi.ryusuke@gmail.com>
Subject: nilfs2: MAINTAINERS: drop unreachable project mirror site
Date: Thu, 8 Feb 2024 18:30:18 +0900

The hosting site where the nilfs project had a mirror site continues to be
in trouble, so we have decided not to use that site.  This will reflect it
in the MAINTAINERS file.

Link: https://lkml.kernel.org/r/20240208093018.6334-1-konishi.ryusuke@gmail.com
Signed-off-by: Ryusuke Konishi <konishi.ryusuke@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 MAINTAINERS |    1 -
 1 file changed, 1 deletion(-)

--- a/MAINTAINERS~nilfs2-maintainers-drop-unreachable-project-mirror-site
+++ a/MAINTAINERS
@@ -15430,7 +15430,6 @@ M:	Ryusuke Konishi <konishi.ryusuke@gmai
 L:	linux-nilfs@vger.kernel.org
 S:	Supported
 W:	https://nilfs.sourceforge.io/
-W:	https://nilfs.osdn.jp/
 T:	git https://github.com/konis/nilfs2.git
 F:	Documentation/filesystems/nilfs2.rst
 F:	fs/nilfs2/
_
